libraries/AC_PID/AC_PID.cpp

```diff
@@ -280,6 +281,8 @@ float AC_PID::update_all(float target, float measurement, float dt, bool limit,
     // scale pd output if required
     P_out *= pd_scale;
     D_out *= pd_scale;
+    // scale i output if required
+    I_out *= i_scale;
```

Comment:
We need to apply this before integration. We want to scale the integration rate, not the output.

Reply:
Fixed